runner/src/server/plugins/engine/pageControllers/SummaryPageController.ts

```diff
@@ -111,7 +111,7 @@ export class SummaryPageController extends PageController {
       this.setFeedbackDetails(summaryViewModel, request);
 
       // redirect user to start page if there are incomplete form errors
-      if (summaryViewModel.result.error) {
+      if (summaryViewModel.result?.error) {
```

Comment:
what was the problem with this?

Reply:
Sorry, missed this. Fixed the issue else where so have removed this